### ISSUE TITLE
Having trouble with my Samsung Galaxy Z Fold 5, which is Linux and arm64

### ISSUE THREAD

**Issue by juliandolby** [diffplug/durian-swt#29]
I have been trying to build WALA (github.com:/wala/WALA) using Andronix on my Samsung Galaxy Z Fold 5, which is arm64 based and looks like Linux when using Andronix.  The build is currently failing when it cannot find the SWT library, which it thinks is org.eclipse.platform:org.eclipse.swt.gtk.linux.unknown:3.124.200.  

WALA builds fine on my arm64-based Mac, and I checked that the aarch64 SWT jar indeed exists.  

WALA uses diffplug, and I think the issue might come from there.  Looking at durian-swt.os/src/main/java/com/diffplug/common/swt/os/OS.java, the code has LINUX_x64 and LINUX_x86. It does know that Apple Silicon is arm64, which might be why my Mac works fine.  But it has nothing for Linux on arm64.

Is there some way to support devices like my Z Fold?

**Comment by juliandolby** [comment]
the function `calculateRunning` looks like it only recognizes Linux as x86 and x64, and it reports unknown when it is running on arm64.

**Comment by nedtwigg** [comment]
Yes, PR's welcome. I think this is where most of the change will need to be, perhaps the only place

- https://github.com/diffplug/durian-swt/blob/main/durian-swt.os/src/main/java/com/diffplug/common/swt/os/OS.java
